cloudnetpy_qc/quality.py

```diff
@@ -1,183 +1,347 @@
-import configparser
+"""Cloudnet product quality checks."""
+import dataclasses
+import datetime
+import logging
 import os
-from typing import Any, List, Tuple
+from dataclasses import dataclass
+from pathlib import Path
+from typing import Dict, List, Optional, Union
 
 import netCDF4
 import numpy as np
+from cfchecker import cfchecks
 from numpy import ma
 
-FILE_PATH = os.path.dirname(os.path.realpath(__file__))
+from . import utils
+from .utils import format_value, str2num
 
+FILE_PATH = os.path.dirname(os.path.realpath(__file__))
 
-QualityResult = Tuple[str, Any, Any]
+METADATA_CONFIG = utils.read_config(f"{FILE_PATH}/metadata_config.ini")
+DATA_CONFIG = utils.read_config(f"{FILE_PATH}/data_quality_config.ini")
 
 
-class Quality:
-    """Class containing quality control routines."""
+@dataclass
+class Product:
```

Comment:
Tämä pitäisi olla `Enum` ennemmin kuin `dataclass`, koska luokkaan ei tallenneta tietoja:

```suggestion
from enum import Enum
class Product(Enum):
```

Reply:
Joo se voisi olla sopivampi tällaiseen käyttöön..